scripts/travis/exit_early

```diff
@@ -11,4 +11,4 @@ then
   fi
 fi
 
-set +v
+set +ev
```

Comment:
`set +e` is necessary at the end of each script to prevent that from bleeding into the parent script. If that happens, then `travis_wait` is terminated before it can print error output.